go-controller/pkg/cni/ovs.go

```diff
@@ -89,6 +89,14 @@ func ovsSet(table, record string, values ...string) error {
 	return err
 }
 
+func GetDatapathType(bridge string) (string, error) {
+	br_type, err := ovsGet("bridge", "br-int", "datapath_type", "")
```

Comment:
replace `br-int` with the bridge argument/parameter

Reply:
ack